### ISSUE TITLE
Cannot Activate MS Office LTSC Professional Plus 2021

### ISSUE THREAD

**Issue by remlei** [SystemRage/py-kms#143]
when attempting to activate, it throws a "0xC004F074" error code.

other activators like vlmcsd works just fine.

**Comment by simonmicro** [comment]
Hey, next time search older issues first: https://github.com/SystemRage/py-kms/issues/140

Please close this.

**Comment by remlei** [comment]
that's not how github works though. can you commit that change so it wont get posted again instead?

**Comment by simonmicro** [comment]
You are right - this is how GitHub works: https://github.com/SystemRage/py-kms/issues/144
